frontend/src/hooks/store/index.ts

```diff
@@ -136,9 +137,12 @@ const initStore: Zustand.StateCreator<Store> = (set, get) => {
         setScheduleRenderingOptions: (options) =>
             set({ scheduleRenderingOptions: options }),
 
+        // TODO: add hide async classes
+        // checking for async -> start-time == end-time || no day (empty array)
         conflictingSectionsOptions: {
             hidden: false,
-            skipSectionsOfSelectedCourse: true,
+            skipSectionsOfSelectedCourse: false,
+            hideAsyncSections: false,
```

Comment:
we want to persist these two options (as they are set in the settings page) but not the master switch of hide conflicting (as it is a special filter from the user's perspective and filters are not saved). So you probably want to split out the `hidden` attribute and add the remaining `conflictingSectionsOptions` to the list of persisted states at the bottom of this file.